Convert fastseg.Rnw to fastseg.Rmd

sonali8434 opened the pull request:
@alexg9010 , kindly review this PR. We use this [Review PR](https://bioconductor.github.io/sweave2rmd/articles/contribute.html#review-a-pull-request) section of our document to review it. We encountered some issues while converting this file.

  * In the original PDF, we have only one plot as the output for this code `segPlot(gr, res, plot.type="s")`. However, in the .Rmd file, if we use the same code, we are getting two plots as output instead of one. The same issue exists with another plot in this file.
  * In the original PDF, the section "Scope and Purpose of this Document" appears above the table of contents. However, it is very difficult to convert it in the .Rmd document. In the HTML document, it appears below the table of contents, just before the Introduction section.

Generally, we give credits to the converter for converting Sweave documents. We ask maintainers if they want us to add ourselves as contributors in the description file and the .Rmd file. I wanted to ask if you would like me to add as a contributor for this conversion. It's totally fine to say no.

After you merge the PR, remember to bump the version in the DESCRIPTION file so that the Bioconductor build system will recognize the new changes and [push the changes to Bioconductor's git repository](https://contributions.bioconductor.org/git-version-control.html#push-to-github-bioc).

Attached:

Original existing [PDF](https://www.bioconductor.org/packages/release/bioc/vignettes/fastseg/inst/doc/fastseg.pdf)
Converted [HTML](https://rpubs.com/Sonali8434/1054340)

sonali8434 committed:
Update DESCRIPTION with BiocStyle, knitr, VignetteBuilder

sonali8434 committed:
Convert file to Rmd

sonali8434 committed:
Remove file.Rnw

sonali8434 committed:
added links to packages

sonali8434 committed:
Remove invisible

sonali8434 committed:
added bioinformatics.csl, some minor changes in .Rmd

sonali8434 committed:
removed invisible from code block

sonali8434 committed:
Added scope of the document

sonali8434 committed:
Removed the author and maintainer line

sonali8434 committed:
fix indents, add email, fix labels

sonali8434 committed:
removed link from DNAcopy objects

alexg9010 commented:
Hi @sonali8434 , 

I finally found time to review your PR and I have to thank you for your effort! The HTML version looks great! 

Thanks for bringing up the issues that occurred during the conversion. 
I think the production of two plots for the HTML should be fine, but moving the "Scope and Purpose of this Document" section above the table of contents would be great. 

Would you have the time to play with this a bit more, I saw from the [Desseq2 Manual](https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html) that it is possible to have content above the TOC. It seems like they are introducing an [abstract](https://github.com/mikelove/DESeq2/blob/24099effd42f31b96bf7c867ea03d7300a6368bc/vignettes/DESeq2.Rmd#L5) into the YAML header of the document. 

Please also feel free to add yourself as a contributor to the package description file.

sonali8434 committed:
Moved scope of document above TOC

sonali8434 committed:
Added sonali as contributor in DESCRIPTION

sonali8434 commented:
> Hi @sonali8434 ,
> 
> I finally found time to review your PR and I have to thank you for your effort! The HTML version looks great!
> 
> Thanks for bringing up the issues that occurred during the conversion. I think the production of two plots for the HTML should be fine, but moving the "Scope and Purpose of this Document" section above the table of contents would be great.
> 
> Would you have the time to play with this a bit more, I saw from the [Desseq2 Manual](https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html) that it is possible to have content above the TOC. It seems like they are introducing an [abstract](https://github.com/mikelove/DESeq2/blob/24099effd42f31b96bf7c867ea03d7300a6368bc/vignettes/DESeq2.Rmd#L5) into the YAML header of the document.
> 
> Please also feel free to add yourself as a contributor to the package description file.

Hello @alexg9010 Thank you for the review, I have moved the scope of document above TOC the new converted [HTML](https://rpubs.com/Sonali8434/1062430) file for your reference. 

And also I have updated the description file. Please review and let me know if you have any more suggestions.

sonali8434 committed:
Removed extra scope of document section.

alexg9010 commented:
Hi @sonali8434 

Thank you for your adjusting the scope section, the final HTML looks awesome! 

I will merge the PR and bump the version. 

Thank you again for your contributions!
